mne/tests/test_epochs.py

```diff
@@ -488,10 +488,11 @@ def test_average_movements():
 
 def _assert_drop_log_types(drop_log):
     __tracebackhide__ = True
-    assert isinstance(drop_log, tuple), "drop_log should be tuple"
+    assert isinstance(drop_log, (tuple, list)), """drop_log should be tuple
```

Comment:
This change we don't want -- `drop_log` should always be immutable, so it must be a tuple (of tuple of str), not list (of tuple of str).